recipes/openfile/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "openfile" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ea4e8f19112049f578ec161b12f70c1113144988c0a488ab173a138b50f37479
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - python 3.6
```

Comment:
Why is the Python fixed? The package looks like it is support 3.x

Reply:
The `build:` section should be empty, this is normally only needed for compilers.